pages/book/operators.mdx

```diff
@@ -98,24 +163,27 @@ Can only be applied to values of type [`Int{:tact}`][int]:
 
 ```tact
 let two: Int = 2;
-two * two;         // 4
-0 * 1_000_000_000; // 0
--1 * 5;            // -5
+two * two;           // 4
+ 0  * 1_000_000_000; // 0
+-1  * 5;             // -5
```

Comment:
why this change?

Reply:
Felt like aligning things around the operator would make for better readability at a glance. May remove this change here and in other places across the file as well

Reply:
Makes sense, will revert those changes